astropy/wcs/wcsapi/low_level_api.py

```diff
@@ -257,16 +258,38 @@ def pixel_bounds(self):
         """
         return None
 
+    @property
+    def pixel_axis_names(self):
+        """
+        An iterable of strings describing the name for each pixel axis.
+
+        If an axis does not have a name, an empty string should be returned.
+        """
+        return [''] * self.pixel_n_dim
+
+    @property
+    def world_axis_names(self):
+        """
+        An iterable of strings describing the name for each world axis.
+
+        If an axis does not have a name, an empty string be returned. Note that
+        these names are just for display purposes and are not standardized.
```

Comment:
Why does the "just for display" purposes appear here and not in `pixel_axis_names`? Isn't that also true for `pixel_axis_names` just without the added bit about "standardized axis types"?

Reply:
Done

Reply:
Done!